app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -32,6 +31,7 @@
     @BindView(R.id.fragment_container)
     ViewPager viewPager;
     private SlidingTabAdapter tabAdapter;
+    private boolean nextVisible = true;
```

Comment:
Let's try to make this work without the need to cache this variable, we want the fragment to hold onto the concept of whether it is currently visible or not and to just inform the view of what to do.